poll.py

```diff
@@ -185,7 +185,7 @@ def vote(self, msg, index):
 
             poll.options[option] += 1
 
-            return '%s:\n%s' % (current_poll, str(poll))
+            # return '%s:\n%s' % (current_poll, str(poll))
```

Comment:
I think this needs *some* feedback at least. Silence will be taken as failure by the bot, unfortunately. Perhaps report the total votes so far be way of one-line feedback?